common/djangoapps/util/testing.py

```diff
@@ -1,9 +1,18 @@
 import sys
 
+from datetime import datetime
+from pytz import UTC
 from mock import patch
 
 from django.conf import settings
 from django.core.urlresolvers import clear_url_caches, resolve
+from openedx.core.djangoapps.course_groups.models import CourseUserGroupPartitionGroup
```

Comment:
It seems a little off to me to use ``openedx`` from ``common``. Could these helpers live in their own Django app?